src/main/scala/tech/cryptonomic/conseil/MainOutputs.scala

```diff
@@ -0,0 +1,169 @@
+package tech.cryptonomic.conseil
```

Comment:
Could we put this file into a specific package? I think the codebase looks much cleaner when only the entrypoints are in the root package.

Reply:
I couldn't come out with a valid package idea, I have now